openapi.yaml

```diff
@@ -503,12 +503,12 @@ paths:
               allOf:
               - $ref: '#/components/schemas/CreditCard'
               - type: object
-              properties:
-                cvv:
-                  type: string
-                  description: >
-                    The Card Verification Value on the back of the card.
-                  example: '123'
+                properties:
```

Comment:
hey heather, I think we need to keep the lines below the `- type: object` line in-line with that array item. For example, this is what it looks like in JSON with the indent i inserted:

```{
   "content": {
      "application/json": {
         "schema": {
            "allOf": [
               {
                  "$ref": "#/components/schemas/CreditCard"
               },
               {
                  "type": "object",
                  "properties": {
                     "cvv": {
                        "type": "string",
                        "description": "The Card Verification Value on the back of the card.\n",
                        "example": "123"
                     }
                  }
               }
            ]
         }
      }
   }
}
```

vs. without: 

```{
   "content": {
      "application/json": {
         "schema": {
            "allOf": [
               {
                  "$ref": "#/components/schemas/CreditCard"
               },
               {
                  "type": "object"
               }
            ],
            "properties": {
               "cvv": {
                  "type": "string",
                  "description": "The Card Verification Value on the back of the card.\n",
                  "example": "123"
               }
            }
         }
      }
   }
}
```

In the without indentation scenario, `properties` isn't in the same dictionary with the previous `type:object`

I've done this indenting a few times in this commit

Reply:
Sounds ok to me